cpp/ql/src/semmle/code/cpp/commons/Alloc.qll

```diff
@@ -11,7 +11,11 @@ predicate allocationFunction(Function f) {
       name = "calloc" or
       name = "realloc" or
       name = "strdup" or
-      name = "wcsdup"
+      name = "wcsdup" or
+      name = "kmem_alloc" or
+      name = "kmem_zalloc" or
+      name = "pool_get" or
+      name = "pool_cache_get"
```

Comment:
These names belong in the section below, under `hasGlobalName`. The section here, under `hasGlobalOrStdName`, is for function names that may exist both at the top level (like `malloc` from `stdlib.h`) and under `std` (like `std::malloc` from `cstdlib`).

Reply:
Alright, that should be better now